paper-api/src/main/java/io/papermc/paper/event/player/PlayerUseBowWithoutProjectileEvent.java

```diff
@@ -0,0 +1,65 @@
+package io.papermc.paper.event.player;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerEvent;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.ApiStatus;
+import org.jspecify.annotations.NullMarked;
+
+/**
+ * Called when a player tries to draw a bow or load a crossbow without having a suitable projectile in their inventory
+ */
+@NullMarked
+public class PlayerUseBowWithoutProjectileEvent extends PlayerEvent {
+
+    private static final HandlerList HANDLER_LIST = new HandlerList();
+
+    private final ItemStack item;
+    private ItemStack projectile;
+
+    @ApiStatus.Internal
+    public PlayerUseBowWithoutProjectileEvent(final Player player, final ItemStack item) {
+        super(player);
+        this.item = item;
+        this.projectile = ItemStack.empty();
+    }
+
+    /**
+     *  Gets the item which the player tries to use
+     *
+     * @return the item
+     */
+    public ItemStack getItem() {
+        return item.clone();
+    }
```

Comment:
Because this event is only specific to boww, I think that this should be getBowItem() or something more specific

Reply:
> tries to draw a bow or load a crossbow

Reply:
We might be able to find a better name I guess, but certainly not related to bows xD

Reply:
I meant bows, which applies to both crossbows/bow. But not sure, maybe leave as is then.

Reply:
Guess something like  `weapon item`?

Reply:
Cant you use anything as "bow" now?

Reply:
Maybe just getWeapon()?